src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -10,19 +10,24 @@
 package container
 
 import (
+	"bytes"
 	"context"
 	"errors"
 	"fmt"
 	"net"
 	"os"
+	"path/filepath"
+	"sort"
 	"strconv"
 	"strings"
 	"sync"
+	textTemplate "text/template"
```

Comment:
yeah I am fine with that realized that the context of file makes it implied. However, I wonder how often is text template being used in go? What I often see that html/template is used so adding that distinction in the variable makes sense if so.

Reply:
I think it might be beneficial if we use both HTML and text templates, in this case, `template` is fine.

Golang Source Code example: https://github.com/golang/go/blob/master/src/text/template/example_test.go